pkgs/by-name/oc/ocis/package.nix

```diff
@@ -0,0 +1,47 @@
+{
+  fetchurl,
+  lib,
+  stdenv,
+  autoPatchelfHook,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "ocis";
+  version = "4.0.6";
+  system =
+    if stdenv.isLinux && stdenv.isx86_64 then
+      "linux-amd64"
+    else if stdenv.isLinux && stdenv.isAarch64 then
+      "linux-arm64"
+    else
+      "";
+
+  src = fetchurl {
+    url = "https://github.com/owncloud/ocis/releases/download/v${version}/${pname}-${version}-${system}";
+
+    sha256 =
+      if stdenv.isLinux && stdenv.isAarch64 then
+        "cb872723f3465619577532681fc5a779117133558c60014a8d543fb48ee0dbb4"
+      else if stdenv.isLinux && stdenv.isx86_64 then
+        "c33e2055359d98c36dfa953b56a7a556d911169db27f0f829c9eb41cf37ff615"
+      else
+        "";
```

Comment:
What happens if this value is reached?

Consider using `builtins.throw` to create a meaningful error message.